packages/ketcher-react/src/script/ui/views/components/Dialog/Dialog.tsx

```diff
@@ -25,6 +25,7 @@ interface DialogProps {
   params: DialogParams
   buttons?: Array<string | React.ReactElement>
   className: string
+  manageFocus?: boolean
```

Comment:
I think name 'manageFocus' is more appropriate for the function than for boolean type